### PR TITLE
Deliverable 2 Initial Code for Review

### PR DESCRIPTION
**Pull request by shashank1994**
Updated with JSON

### CODE REVIEW

**Comment by kwhitehall** [comment]
Hey @shashank1994, @mich326, and @kushaank this is a great first attempt! Great news is (drum roll) it more or less works :+1: So, let's clean it up now. 
1. Please read up on classes (http://www.tutorialspoint.com/python/python_classes_objects.htm). We want to make this a class with all these methods, as they are associated with the same journal. 
2. Also, add a doc string for each method
3. +1 on the try blocks as it relates to populating the JSON. It would be prudent to also put in some try blocks in some of the other methods for graceful exit incase for e.g.  a path isn't found.

**Comment by kwhitehall** [comment]
1. A method to create a seedlist (seed.txt) that is updated during the crawl. The seedlist should contain a url per a line.

**Comment by lewismc** [comment]
REST API?
https://github.com/chrismattmann/nutch-python

**Comment by kwhitehall** [comment]
@lewismc can you please expound?

**Comment by lewismc** [comment]
The Python API enables injection of URLs via the REST API. If you wish to add URLS, then all of this stuff can be done RESTfuly. I can come and get more context from you later today @kwhitehall

**Comment by kwhitehall** [comment]
Yes @lewismc, you're jumping the gun on the next deliverable. Re this one, getting the seedlist is sufficient. And yeap... we should chat.

**Comment by chrismattmann** [comment]
Yes use rest API nutch-Python ftw

File: testSelenium.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+		filenameJSON = os.getcwd()+'/jsonFilesAMS/'+ json_dict['id'].split('://')[1].replace('/','-')+'.json'
+		with open(filenameJSON, 'w+') as f:
+			json.dump(json_dict, f)
+
+	def get_seed_info(self, currLink):
+		'''
+			Purpose: To compile a list of all the URLs explored in a consolidated text file. 
+		'''
+		file = open('seedList.txt', 'a');
```
Comment (kwhitehall) [inline testSelenium.py:195]: be careful of using special keywords for file names. e.g file
also when explicitly opening a file handler, remember to close it!

**Pull request merged**